Automattic/wp-calypso
# GuidedTours: Frontend preview

_edits by @mcsf_

~~Based off #4391~~
Intends to close #4380

### TODO
- [x] Might be worth looking into `DesignPreview` for this.
- [x] Properly hide/disable `primary` when showing preview
- [x] Properly position preview relative to sidebar (currently breaks with resizes)
- [ ] z-index: Position preview below `.environment-badge`
- [x] Be more defensive, codewise, when no site selected
- [x] Hide the sidebar at first

### Current state
![calypso-guidestours-160408](https://cloud.githubusercontent.com/assets/150562/14394357/c045ed0e-fdc3-11e5-8a82-9c2a58abfbc5.gif)
(Previously: [#](https://cloud.githubusercontent.com/assets/215074/14205997/bbeffee6-f807-11e5-9c1d-2997f5c79bdb.gif))


### To test
- Add `?tour=main` to any page **where a site has been selected**
- Reload the page, hopefully see a preview and a tour
- Play around a bit
